### PR TITLE
Fix unit tests for jfa 0.5.3 and R 4.0.5

### PR DESCRIPTION
**Pull request by koenderks**

### CODE REVIEW

**Comment by TimKDJ** [comment]
Partial fix for https://github.com/jasp-stats/INTERNAL-jasp/issues/1358
@vandenman can you restart the unit tests after you push the changed workflow

File: DESCRIPTION [hunk 1]
```diff
@@ -16,9 +16,9 @@ Imports:
   ggrepel,
   jaspBase,
   jaspGraphs,
-  jfa (>= 0.5.0),
```
Comment (TimKDJ) [inline DESCRIPTION:19]: You need to specify `jfa` under `Imports` and `Remotes`: https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html

File: tests/testthat/test-auditbayesianplanning.R [hunk 2]
```diff
@@ -183,12 +183,12 @@ test_that("Across Expected Errors (Current: 0) plot matches 3", {
...
-			 0.8977, "Prior", 1927, "beta-binomial(N = 9987, <unicode><unicode> = 1, <unicode><unicode> = 14)",
-			 0.955981093321683, 0.0440189066783169, 0, 21.7175110755894,
-			 0.1927, "Expected posterior", -7573, "", 4.77799445992478, 0.0550291357112318,
-			 -523, 86.8266309868567, "", "Expected shift"))
+		list(9501, "beta-binomial(N = 10000, <unicode><unicode> = 1, <unicode><unicode> = 1)",
+			 0.200079992000814, 0.799920007999186, "", 0.250125000000022,
+			 "", "Prior", 1926, "beta-binomial(N = 9987, <unicode><unicode> = 1, <unicode><unicode> = 14)",
+			 0.956181004359911, 0.0438189956400888, 0, 21.8211529130788,
+			 1926, "Expected posterior", -7575, "", 4.77899361549366, 0.054779221924567,
+			 "", 87.2409911567292, "", "Expected shift"))
```
Comment (TimKDJ) [inline tests/testthat/test-auditbayesianplanning.R:191]: So were the old results wrong?

File: tests/testthat/test-auditclassicalbenfordslaw.R [hunk 3]
```diff
@@ -1,13 +1,13 @@
 context("[Audit] Benford's Law")
 
-options <- jaspTools::analysisOptions("auditClassicalBenfordsLaw")
+options <- analysisOptions("auditClassicalBenfordsLaw")
```
Comment (TimKDJ) [inline tests/testthat/test-auditclassicalbenfordslaw.R:3]: Nitpick: if you keep the `jaspTools::` then you don't need `library(jaspTools)` to test the package.

File: tests/testthat/test-auditbayesianplanning.R (hunk 2 above)
Comment (koenderks) [reply tests/testthat/test-auditbayesianplanning.R:191]: Yes there was an error in the calculation for the beta-binomial mode.

File: R/auditCommonFunctionsBayesian.R [hunk 4]
```diff
@@ -48,8 +48,8 @@
     lowerBound <- qbeta(lowerBoundConfidence, shape1 = alphaParameterPosterior, shape2 = betaParameterPosterior)
     upperBound <- qbeta(upperBoundConfidence, shape1 = alphaParameterPosterior, shape2 = betaParameterPosterior)                                        
   } else if(likelihood == "hypergeometric"){
-    lowerBound <- jfa:::.qBetaBinom(lowerBoundConfidence, N = N - n + k, shape1 = alphaParameterPosterior, shape2 = betaParameterPosterior)	/ N
```
[the remark this answers is not in the document]
Comment (koenderks) [reply R/auditCommonFunctionsBayesian.R:51]: The calculation now takes a different input yes.

File: .github/workflows/unittests.yml [hunk 5]
```diff
@@ -12,8 +12,8 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: windows-latest, r: "3.6.1"}
-          - {os: macOS-latest,   r: "3.6.1"}
+          - {os: windows-latest, r: "4.0.5"}
+          - {os: macOS-latest,   r: "4.0.5"}
```
Comment (TimKDJ) [inline .github/workflows/unittests.yml:17]:
```suggestion
- {os: ubuntu-20.04, r: "4.0.5"}
```
You can also test it on linux now

**Pull request merged**